go/apps/api/openapi/spec/paths/chproxy/metrics/ChproxyMetricsResponseBody.yaml

```
@@ -0,0 +1,8 @@
+type: object
+required:
+  - status
+properties:
+  status:
+    type: string
+    description: Processing status
+    example: "OK"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)